tests/checkLibraries.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+lambdapi='dune exec -- lambdapi check'
+TIMEFORMAT="%Es"
+out=/tmp/lambdapi.output
+
+checkoutLib() {
+    git clone $1 $2
+}
+
+checkLib() {
+    for f in $(find $1 -name *.lp)  $(find $1 -name *.dk)
+    do
+        echo lambdapi check $options $f ...
+        $lambdapi "$f" > $out 2>&1
+        if test $? -ne 0; then cat $out; exit 1; fi;
+    done
+}
+
+checkoutLib $1 $2
+options='-w'
```

Comment:
I can move it to line 3 but I believe that since it's only used by checkLib that is called on line 22, it should work too.
![image](https://github.com/Deducteam/lambdapi/assets/40537601/ce05df6f-e9de-47e3-8470-77b33c05c87d)